components/brave_new_tab_ui/brave_new_tab.tsx

```diff
@@ -21,29 +22,35 @@ import 'emptykit.css'
 import '../fonts/poppins.css'
 import '../fonts/muli.css'
 
-window.cr.define('brave_new_tab', function () {
-  'use strict'
-
-  function initialize () {
-    render(
-      <Provider store={store}>
-        <ThemeProvider theme={Theme}>
-          <App />
-        </ThemeProvider>
-      </Provider>,
-      document.getElementById('root'))
-    window.i18nTemplate.process(window.document, window.loadTimeData)
-  }
-
-  function statsUpdated () {
-    const actions = dataFetchAPI.getActions()
-    actions.statsUpdated()
-  }
-
-  return {
-    initialize,
-    statsUpdated
-  }
-})
-
-document.addEventListener('DOMContentLoaded', window.brave_new_tab.initialize)
+async function initialize () {
+  render(
+    <Provider store={store}>
+      <ThemeProvider theme={Theme}>
+        <App />
+      </ThemeProvider>
+    </Provider>,
+    document.getElementById('root')
+  )
+  window.i18nTemplate.process(window.document, window.loadTimeData)
+  handleAPIEvents()
+  await updatePrefs()
+}
+
+async function updatePrefs () {
```

Comment:
nit: can we replace `prefs` with `preferences` everywhere?

Reply:
👍

Reply:
This is fixed, thanks.